src/pages/stablecoins.tsx

```diff
@@ -0,0 +1,753 @@
+                ml={{ base: "auto", lg: 0 }}
+                mr={{ base: "auto", lg: 8 }}
+              >
+                <H2 fontSize="2rem" mt={0} mb={2}>
+                  {t("page-stablecoins-usdc-banner-title")}
+                </H2>
+                <Text fontSize="xl" lineHeight={1.4} color="text200">
+                  {t("page-stablecoins-usdc-banner-body")}
+                </Text>
+                <Flex direction="column">
+                  <Box>
+                    <ButtonLink
+                      mb={4}
+                      mr={4}
+                      to="https://matcha.xyz/markets/ETH/USDC"
+                    >
+                      {t("page-stablecoins-usdc-banner-swap-button")}
+                    </ButtonLink>
+                  </Box>
+                  <Box>
+                    <ButtonLink
+                      variant="outline"
+                      to="https://www.coinbase.com/usdc"
+                    >
+                      {t("page-stablecoins-usdc-banner-learn-button")}
+                    </ButtonLink>
+                  </Box>
+                </Flex>
+              </Box>
+              <Image
+                src={usdcLargeImg}
+                alt={t("page-stablecoins-usdc-logo")}
+                backgroundSize="cover"
+                backgroundRepeat="repeat"
+                alignSelf="center"
+                flex="1"
+                maxWidth={{ base: "96px", sm: "160px", md: "240px" }}
+                minWidth={{ base: "96px", sm: "160px" }}
+                mb={{ base: 8, md: 0 }}
+              />
+            </EditorsChoice>
+          </SimpleGrid>
+          <H3>
+            {t("page-stablecoins-top-coins")}
+            <Tooltip content={tooltipContent}>
+              <Icon ml={2} fill="'text" name="info" boxSize={4} />
+            </Tooltip>
+          </H3>
+          <Text>
+            {t("page-stablecoins-top-coins-intro")}{" "}
+            {t("page-stablecoins-top-coins-intro-code")}
+          </Text>
+        </Box>
+        <Box px={8} py={4} width="full" overflowX="scroll">
+          <StablecoinsTable
+            columns={tableColumns}
+            content={markets}
+            hasError={marketsHasError}
+          />
+        </Box>
+      </Box>
+      <Content id="explore">
+        <H2>{t("page-stablecoins-get-stablecoins")}</H2>
+      </Content>
+      <Flex
+        alignItems="center"
+        w="full"
+        pl={{ base: "0rem", lg: "2rem" }}
+        pr={{ base: "0rem", lg: "2rem" }}
+        pb={16}
+      >
+        <StablecoinAccordion />
+      </Flex>
+      <Divider />
+      <Content>
+        <CalloutBanner
+          mt={8}
+          mb={16}
+          mx={0}
+          titleKey={t("page-stablecoins-stablecoins-dapp-callout-title")}
+          descriptionKey={t(
+            "page-stablecoins-stablecoins-dapp-callout-description"
+          )}
+          image={dogeComputerImg}
+          maxImageWidth={600}
```

Comment:
<img width="736" alt="image" src="https://github.com/ethereum/ethereum-org-fork/assets/54227730/e6eaac46-c752-4326-be4e-3a44ad2668e1">

Getting a horizontal overflow from this image on mobile

Reply:
Looks like this is related to the CalloutBanner component, since the other usage of this component is also broken in this PR:

<img width="558" alt="image" src="https://github.com/ethereum/ethereum-org-fork/assets/54227730/a61a8c2c-2371-4cbd-8329-b9fd4f9a7ecc">

vs the `dev` brnch:

<img width="577" alt="image" src="https://github.com/ethereum/ethereum-org-fork/assets/54227730/feed3343-2196-404a-8b6e-7ea4815fb099">